roles/core/repositories_client/tasks/centos_7.yml

```diff
@@ -13,13 +13,32 @@
 
 - name: Set baseurl prefix
   set_fact:
-    baseurl_prefix: "http://{{ networks[j2_node_main_network]['services_ip']['repository_ip'] }}/repositories/{{ equipment_profile['operating_system']['distribution'] }}/{{ equipment_profile['operating_system']['distribution_version'] }}/{{ equipment_profile['hardware']['cpu']['architecture'] }}/"
+    baseurl_prefix: "http://{{ networks[j2_node_main_network]['services_ip']['repository_ip'] }}/repositories/{{ equipment_profile['operating_system']['distribution'] }}/$releasever/$basearch/"
+
+- name: Force $releasever
```

Comment:
I like this.
Allows to use remote updates without updating to next minor in case of need. Then simply remove lock with next task.

Would this lock work if user do a createrepo on updates/erratas? If not, would be nice to add this warning in the documentation.

Reply:
Also why do you force 'http', we should open it to https to by using a variable set by default on http if you like, but pretty sure one day https will be needed too.

Reply:
While I like the idea, can this http/https feature request be postponed to another PR :-)

Reply:
I agree on this. Https will have to be handled at some point, but please later.